### PR TITLE
Fix sphinx documentation

### PR DESCRIPTION
**Pull request by CharlieL7**
Fixes some bugs in the documentation

### CODE REVIEW

**Review by TedThemistokleous — APPROVED** [review]

**Review by kahmed10 — APPROVED** [review]

File: docs/driver/compile.rst [hunk 1]
```diff
@@ -1,5 +1,3 @@
-.. include:: ./driver/read.rst
```
Comment (pfultz2) [inline docs/driver/compile.rst:1]: Why is this removed here?

Comment (CharlieL7) [reply docs/driver/compile.rst:1]: Leads to a weird error with the current documentation where it tries to find the read.rst at `../../driver/driver/read.rst`.

Comment (pfultz2) [reply docs/driver/compile.rst:1]: Maybe it should just be `read.rst`.

**Review by umangyadav — APPROVED** [review]

File: docs/driver/compile.rst (hunk 1 above)
Comment (CharlieL7) [reply docs/driver/compile.rst:1]: Doesn't work, then the `make doc` complains about:
```
../../docs/driver/compile.rst:1: CRITICAL: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: '../../docs/read.rst'.
```
